lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -2158,6 +2190,23 @@ struct DotOpConversionHelper {
     return Type{};
   }
 
+  Type getLoadElemTy() {
+    switch (mmaType) {
+    case TensorCoreType::FP32_FP16_FP16_FP32:
```

Comment:
Here the first `FP32` means the output, the second and third `FP16` means the type of A/B, `FP32` means the type of accumulator? Or the first `FP32` means the data type of the `C` operand？

Reply:
I think, the first FP32 is the result type and the last one is `$c`'s type.

$d = $a * $b + $c